src/components/data/EntitiesContent.js

```diff
@@ -284,9 +284,8 @@ const EntitiesContent = ({
   } = useColumnProvenance(workspace, entityKey);
   const [showColumnProvenance, setShowColumnProvenance] = useState(undefined);
 
```

Comment:
`EntitiesContent` and `EntitiesContent.test` are here because I branched off of https://github.com/DataBiosphere/terra-ui/pull/4126 ; please ignore them for now.